src/main/java/core/basesyntax/service/filewriter/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.filewriter;
+
+import core.basesyntax.service.exeptions.FileWriterExeption;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void writeToFile(List<String> report, String filePath) throws IOException {
```

Comment:
throws IOException why do u use throws IOException signature and then have a try/catch block?

Reply:
agree

Reply:
removed